### ISSUE TITLE
Training accuracy of train_cifar10.py

### ISSUE THREAD

**Issue by ThornsOoO** [jordan-g/Burstprop#4]
Hi jordan:
I‘m very interested in your article and tried to run train_ Cifar10.py, but the accuracy is always about 10%. I remember that in "burst dependent synaptic plastic can coordinate learning in hierarchical circuits", the accuracy of burstprop on cifar10 is close to  BackProp. 
I'm very confused and don't know what's wrong.
I didn't make any additional changes to the super parameters, just python3 train_ Cifar10.py

NVIDIA Tesla K80  cuda11

Simulation run @ 10:01AM on November 26, 2021
Number of epochs: 400
Batch size: 32
Using validation set: False
Feedforward learning rates: [0.01, 0.01, 0.01, 0.01, 0.01]
Feedback weight initialization standard deviation: 1.0
Momentum: 0.9
Weight decay: 1e-06
Output layer baseline burst probability: 0.2
Using backprop: False
Feedback weight learning: True
Recurrent input: True
Recurrent weight initialization standard deviation: 0.01
Recurrent weight learning: True
Recurrent weight learning rate: 0.0001
Using node pertubation: False
Sequential node pertubation: False
Perturb only single units: False
Mean of node pertubation noise: 0.0
Standrd deviation of node pertubation noise: 0.001

**Comment by jordan-g** [comment]
Hi ThornsOoO,

My apologies for the late reply. The default parameters will not give good performance, here are parameters that will allow you to replicate the CIFAR-10 results from the paper:

n_epochs 400
batch_size 32
hidden_lr 0.7
output_lr 0.006
weight_fa_std 0.03
momentum 0.9
weight_decay 1e-6
p_baseline 0.2
use_backprop False
weight_fa_learning True
recurrent_input True
weight_r_range 0.01
weight_r_learning True
recurrent_lr 0.0001
use_node_pertubation False

**Comment by ThornsOoO** [comment]
Hi jordan:
Thank you very much for your reply.I have tried the following parameters you provided, and the effect is very good.
To be honest, I tried to modify cifar10 Py for text classification, but unfortunately there is no learning effect. It may be the problem of network structure. Now I try to transplant your method into the ppo2 algorithm. Do you think it is feasible?
I'm very sorry, because I'm just beginning to contact the knowledge of Depth learning, and there may be problems in the description
